source/indexes/compound-index.txt

```diff
@@ -0,0 +1,99 @@
+.. _cpp-compound-index:
+
+================
+Compound Indexes
+================
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+.. facet::
+   :name: genre
+   :values: reference
+ 
+.. meta::
+   :keywords: index, query, optimization, efficiency
+
+Overview
+--------
+
+**Compound indexes** are indexes that hold references to multiple
+fields within a collection's documents. These indexes improve multi-field query and
+sort performance.
+
+To create a compound index, call the ``create_index()`` method and specify a document containing the following information: 
+
+- Fields on which to create the index
+- Sort order for the indexed values, ``1`` for ascending or ``-1`` for descending.
```

Comment:
S: I'd separate the second bullet into 2 sentences. Also, since the second one has periods, [add a period](https://www.mongodb.com/docs/meta/style-guide/style/lists/#list-items) to the end of the first bullet point
```suggestion
- Fields on which to create the index.
- Sort order for the indexed values. Use ``1`` for ascending or ``-1`` for descending.
```